### PR TITLE
Fix sign if delz_inc if write_fv3_incr=T

### PR DESCRIPTION
**Pull request by jswhit**

### CODE REVIEW

**Review by CoryMartin-NOAA — APPROVED** [review]
Thank you @jswhit (and to @CatherineThomas-NOAA for diving down) I (at least for now) am confident that this looks right, finally...

File: src/enkf/gridio_gfs.f90
Comment (RussTreadon-NOAA) [inline src/enkf/gridio_gfs.f90:2883]: `krev` is not declared in `subroutine writegriddata`.   This causes compilation to fail.

**Review by RussTreadon-NOAA — COMMENTED** [review]
Compilation fails due to `krev` being undefined in `subroutine writegriddata` in file `src/enkf/gridio_gfs.f90`

**Comment by MichaelLueken-NOAA** [comment]
@RussTreadon-NOAA @CoryMartin-NOAA @CatherineThomas-NOAA @jswhit - Given feedback in issue #117, should these changes go into the master or go into release/gfsda.v16.1.0?  Only changes in the release branches will be implemented, so it would probably be best to either add these to release/gfsda.v16.1.0 along with the commercial GPSRO update, or add it to release/gfsda.v16.0.0 (or a separate branch from this release branch).  These changes would then be merged to the master along with the release branch.  Thoughts?

**Comment by CoryMartin-NOAA** [comment]
I don't think @dtkleist has made a decision yet on if/when these fixes would be implemented. So probably not into release/gfsda.v16.1.0 at least not yet.

**Comment by dtkleist** [comment]
@MichaelLueken-NOAA -- @CoryMartin-NOAA is correct...operational path forward on this has not yet been decided until further testing and coordination is completed. We should definitely not merge this into release/gfsda.v16.1.0 until that pathway is cleared & approved (assuming/if we want to go that route to begin with).

**Comment by MichaelLueken-NOAA** [comment]
These changes have been merged to the authoritative repo at 51a6564.  Closing this PR (and issue #117).

**Pull request closed without merging**
